client/src/components/IconButton.js

```diff
@@ -17,6 +18,7 @@ export const IconButton = styled.button`
   align-items: center;
   width: ${(props) => (props.nav ? "48px" : "auto")};
   height: ${(props) => (props.nav ? "48px" : "auto")};
```

Comment:
When you set multiple rules you can do it like this
```suggestion
import {css} from 'styled-components'
//...

${(props) => props.nav 
  ? css`
    width: 48px;
    height: 48px;
  `
  : css`
    width: auto;
    height: auto;
  `
`}
```